src/shared/header/header.component.ts

```diff
@@ -84,12 +93,16 @@ export class HeaderComponent implements OnInit, OnChanges, OnDestroy, AfterViewI
     this.mapComponent = this.activatedRoute.snapshot.url[0].path === 'map';
   }
 
-  /* tslint:disable-next-line */
-  public ngAfterViewInit(): void {}
+  public ngAfterViewInit(): void {
+    this.incomeContainer = this.element.querySelector('.income-title-container') as HTMLElement;
```

Comment:
We have a hole section in performance iteration to replace querySelectors with ViewChild let do this during it